### ISSUE TITLE
wiki: CodeReviewComments lacks entry on commit-message style

### ISSUE THREAD

**Issue by RalphCorderoy** [golang/go#26744]
https://go-review.googlesource.com/c/text/+/127176 is a common case of the phrase after the sentence not completing the sentence "This change modifies Go to _____".  Instead of pointing them at a CodeReviewComments wiki entry, I have to dig up https://golang.org/doc/contribute.html#commit_messages

A wiki entry would place the topic alongside the others, meaning reviewers are more likely to notice and refer to it.  The new entry could still refer to contribute.html#commit_messages for the full detail.

**Comment by ALTree** [comment]
> Instead of pointing them at a CodeReviewComments wiki entry...

... you should point them to https://github.com/golang/go/wiki/CommitMessage

**Comment by RalphCorderoy** [comment]
Thanks, I didn't know of that page.  It has quite a bit of overlap with doc/contribute.html that it refers to; more so that what I was envisaging for CodeReviewComments.  But it's not a replacement so both need to be read, overlap included.  Perhaps this means there's still room for improvement between the three of them.  If not, that's fine as I know now to point to CommitMessage.

**Comment by ALTree** [comment]
I'm still not sure what you're suggesting to do. At first you were suggesting to create a new wiki page, with a full recipe for a good commit message, so we can point that page out to new contributors. That page already exists, so... done? But then you wrote:

> Perhaps this means there's still room for improvement between the three of them

What kind of "improvement"? More documentation? Less? Less overlap?

**Comment by RalphCorderoy** [comment]
No, at first I was suggesting to create a new item on the existing wiki page https://github.com/golang/go/wiki/CodeReviewComments as it's a common resource for reviewers to reference to authors. By being on that page, it would be seen by reviewers and used by them, unlike the existing single page https://github.com/golang/go/wiki/CommitMessage

Separately, the two existing resources seem to have unnecessary overlap that would be tedious for a new author to wade through. This suggests to me they could possibly be rationalised, and that might include a new item on the CodeReviewComments page.

**Comment by ianlancetaylor** [comment]
I see CodeReviewComments and CommitMessage as somewhat different.  CodeReviewComments is intended to be a general resource that applies to all Go code written by everyone everywhere.  CommitMessage is specific to the code review style used by the Go project.  I think there is likely to some benefit to not confusing the two.
